ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -241,14 +241,17 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+    /// Making these stack views as public so as to access the arranged subviews of these stack view.
+    /// One use case is to access the arranged subview to show the tooltip.
+    @objc public let rightBarButtonItemsStackView = UIStackView()
```

Comment:
I don't think we should directly expose like that. then there is a risk that rightbarbuttonitem and the stack view can be out of sync. shouldn't it be a helper function that returns the UIView similar to how don't expose avatar view directly but give enough functions

Reply:
My bad!! This totally makes sense. I have created a helper function to get the bar button item view with a particular tag.